python_modules/libraries/dagster-dbt/dagster_dbt/asset_defs.py

```diff
@@ -977,8 +975,6 @@ def get_auto_materialize_policy(
         use_build_command=use_build_command,
         partitions_def=partitions_def,
         partition_key_to_vars_fn=partition_key_to_vars_fn,
-        node_info_to_freshness_policy_fn=node_info_to_freshness_policy_fn,
```

Comment:
where are these going?

Reply:
They were unused arguments. They have been subsumed in `dagster_dbt_translator`.